Packs/PaloAltoNetworks_IoT3rdParty/Scripts/SendAllPANWIoTAlertsToSIEM/SendAllPANWIoTAlertsToSIEM.py

```diff
@@ -0,0 +1,109 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+count = 0
+res = []
+offset = 0
+PAGE_SIZE = 1000
+
+SIEM_INSTANCE = "PANW IoT 3rd Party SIEM Integration Instance"
+PANW_IOT_INSTANCE = "PANW IoT 3rd Party Integration Instance"
+
+'''
+returns a status and message back to cloud
+'''
```

Comment:
please consider refactoring with a `main()` entry point, it is quite hard to track this code.

Reply:
added main()